docker-compose.yml

```diff
@@ -29,5 +29,8 @@ rq_scheduler:
   links:
     - db
     - redis
+    # django-rq doesn't support rqscheduler `retry` mode yet
+    # so we need to use the original startup script, which retrieves
+    # the
   command:
-    python manage.py rqscheduler
+    rqscheduler --url=redis://redis:6379
```

Comment:
I would agree if this was a long term thing, but at the end it's gonna be there (hopefully) for a short time, until the patch gets merged upstream and the new parameter is added to django-rq as well.